src/pages/st-payment/st-payment.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import Layout from '../../components/layout/layout';
+import environment from '../../environment/environment';
+import '../../../static/libraries/st.css';
+import { Heading } from '../../components/heading/heading';
+
+const newJwt =
+  'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJhbTAzMTAuYXV0b2FwaSIsImlhdCI6MTU5MDE1NDgwNy4wNzQ5ODg2LCJwYXlsb2FkIjp7Im1haW5hbW91bnQiOiIyMC4wMCIsImFjY291bnR0eXBlZGVzY3JpcHRpb24iOiJFQ09NIiwiY3VycmVuY3lpc28zYSI6IkdCUCIsInNpdGVyZWZlcmVuY2UiOiJ0ZXN0X2phbWVzMzg2NDEiLCJsb2NhbGUiOiJlbl9HQiJ9fQ.eXAxDB5yOaM-63k6tf0634ojvQo7zDuuXeAKmP3DtGw';
+
+class StPayment extends Component {
+  componentDidMount() {
+    const self = this;
+
+    self.loadConfig().then(function(config) {
+      self.loadST(config);
+    });
+  }
+
+  updateJwt() {
+    document.getElementById('example-form-amount').addEventListener('input', () => this.instance.updateJWT(newJwt));
+  }
+
+  loadST(config) {
+    const { components, applePay, visaCheckout } = config;
+    this.instance = SecureTrading(config);
+
+    this.instance.submitCallback = data => {
+      console.error(`This is what we have got after submit ${JSON.stringify(data)}`);
+    };
+
+    this.instance.successCallback = () => alert('Success alert');
+    this.instance.errorCallback = () => alert('This is error message');
+    this.instance.Components(components);
+    this.instance.ApplePay(applePay);
+    this.instance.VisaCheckout(visaCheckout);
+    this.updateJwt();
```

Comment:
`updateJwt()` => `initUpdateJwtListener()` or something like that 😉

Reply:
good point, changed